doc/astspec.txt

```diff
@@ -223,6 +287,9 @@ AST:
 .. code-block:: nim
   nnkDotExpr(nnkIdent(!"x"), nnkIdent(!"y"))
 
+If you use Nim's flexible calling syntax (as in ``x.len()``), the result is the 
+same as above but wrapped in an ``nnkCall``.
```

Comment:
This is only the case if you put parentheses at the end, right? I should mention that, I guess.